JavaComparator/Comparator.java

```diff
@@ -0,0 +1,133 @@
+import java.util.ArrayList;
+
+public class Comparator {
+    String file_orig, file_tripped;
```

Comment:
Change all the variable names in all java files so that they are clear what they are referring to -example: originalFilePath, roundtrippedFilePath

Reply:
+1